lib/sol/SolRpc.js

```diff
@@ -594,28 +751,92 @@ class SolRPC {
   }
 
   /**
-   * Converts a base64 encoded string to a Uint8Array.
-   * 
-   * @param {string} str - The base64 encoded string to convert.
-   * @returns {Uint8Array} The converted Uint8Array.
-   */
+ * Converts a base64 encoded string to a Uint8Array.
+ * 
+ * @param {string} str - The base64 encoded string to convert.
+ * @returns {Uint8Array} The converted Uint8Array.
+ */
   base64ToUint8Array(str) {
-    // Decode the base64 string to a Buffer
+  // Decode the base64 string to a Buffer
     const buffer = Buffer.from(str, 'base64');
     // Convert the Buffer to a Uint8Array
     const uint8Array = new Uint8Array(buffer);
     return uint8Array;
   }
 
   /**
-   * Converts a Uint8Array to a base64 encoded string.
-   * 
-   * @param {Array} arr - The Uint8Array to convert.
-   * @returns {string} The converted base64 encoded string .
-   */
+ * Converts a Uint8Array to a base64 encoded string.
+ * 
+ * @param {Array} arr - The Uint8Array to convert.
+ * @returns {string} The converted base64 encoded string .
+ */
   uint8ArrayToBase64(arr) {
     return this.toBuffer(arr).toString('base64');
   }
-}
 
-module.exports = SolRPC;
\ No newline at end of file
+  async #getLatestBlockhash() {
+    const { value } = await this.rpc.getLatestBlockhash().send();
+    return value;
+  }
+
+  #bigIntToNumber(number) {
+    return Number(number);
+  }
+
+  /**
+   * @param {import('@solana/web3.js').TransactionMessageBytes} messageBytes 
+   * @returns {import('@solana/web3.js').TransactionMessageBytesBase64}
+   */
+  #transactionMessageBytesToTransactionMessageBytesBase64(messageBytes) {
+    return Buffer.from(messageBytes).toString('base64');
```

Comment:
That's one heck of a function name. In fact, it's shorter and easier to read
```
Buffer.from(messageBytes).toString('base64')
```
than
```
transactionMessageBytesToTransactionMessageBytesBase64
```

Kill it.